pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -22,7 +24,19 @@ class Range(Module):
   is_original_torch_module = False
 
   def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) -> Dict[str, Any]:
-    return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+    if isinstance(limit, SizeValue) and limit.dim_tag.dimension is None:
```

Comment:
When we change `Naming._make_tensor` as discussed, we would not get `SizeValue` here but instead we would get a `Tensor`.